Documentation/Debugger/gdb/Windows Subsystem for Linux.md

```diff
@@ -1,91 +1 @@
-
-**NOTE: Applications will need to be compiled in the `Windows Subsystem for Linux (WSL)` prior to debugging.**
-
-### Example `launch.json` for Launching
-
-In the following example, I have a local drive, `Z:\` that has my source code within windows for an app called _kitchensink_. I have set up the `"program"` and `"cwd"` paths to point to the directory within `WSL`. I have set up the `"pipeTransport"` to use `bash.exe`. I have also set up a `"sourceFileMap"` to have everything that is returned by `gdb` that starts with `/mnt/z` to point to `Z:\\` in Windows.
-
-```json
-{
-    "name": "C++ Launch",
-    "type": "cppdbg",
-    "request": "launch",
-    "program": "/mnt/z/Bash/kitchensink/a.out",
-    "args": ["-fThreading"],
-    "stopAtEntry": false,
-    "cwd": "/mnt/z/Bash/kitchensink",
-    "environment": [],
-    "externalConsole": true,
-    "windows": {
-        "MIMode": "gdb",
-        "setupCommands": [
-            {
-                "description": "Enable pretty-printing for gdb",
-                "text": "-enable-pretty-printing",
-                "ignoreFailures": true
-            }
-        ]
-    },
-    "pipeTransport": {
-        "pipeCwd": "",
-        "pipeProgram": "c:\\windows\\sysnative\\bash.exe",
-        "pipeArgs": ["-c"],
-        "debuggerPath": "/usr/bin/gdb"
-    },
-    "sourceFileMap": {
-        "/mnt/z": "z:\\"
-    }
-}
-```
-
-### Example `launch.json` for Attaching to an Existing Process
-
-This configuration similar to the launch process above. I have chosen to start the same application above from the Bash command line and now I want to attach to it for debugging. I have changed the `"processID"` to use the remote process picker by specifying the command `"${command:pickRemoteProcess}"` and set up the same `"sourceFileMap"`. When I press <kbd>F5</kbd> to attach, I get a picker drop down showing the running processes within `WSL`. I can scroll or search for the process I want to attach to and start debugging.
-
-```json
-{
-    "name": "C++ Attach",
-    "type": "cppdbg",
-    "request": "attach",
-    "program": "/mnt/z/Bash/kitchensink/a.out",
-    "processId": "${command:pickRemoteProcess}",
-    "windows": {
-        "MIMode": "gdb",
-        "setupCommands": [
-            {
-                "description": "Enable pretty-printing for gdb",
-                "text": "-enable-pretty-printing",
-                "ignoreFailures": true
-            }
-        ]
-    },
-    "pipeTransport": {
-        "pipeCwd": "",
-        "pipeProgram": "c:\\windows\\sysnative\\bash.exe",
-        "pipeArgs": ["-c"],
-        "debuggerPath": "/usr/bin/gdb"
-    },
-    "sourceFileMap": {
-        "/mnt/z": "z:\\"
-    }
-}
-```
+The [documentation for GCC on Windows Subsystem for Linux (WSL)](https://code.visualstudio.com/docs/cpp/config-wsl) has moved to [Visual Studio Code Docs](https://code.visualstudio.com/docs).
```

Comment:
Let's do one link per file to avoid potential confusion.  We don't need to link to the root of the documentation.